blog/tidy-market-microstructure/index.qmd

```diff
@@ -0,0 +1,1674 @@
+---
+title: "Tidy Market Microstructure"
+author:
+  - name: Björn Hagströmer
+    url: https://hagstromer.org/
+    affiliations:
+      - name: Stockholm Business School
+  - name: Niklas Landsberg
+    url: https://www.kuleuven.be/wieiswie/en/person/00167120
+    affiliations:
+      - name: KU Leuven 
+date: "2023-02-15"
+description: A beginner's guide to market quality measurement in high-frequency data
+image: thumbnail.png
+image-alt: Cartoon of robots and humans trying to make sense of green stock price charts on black background. Created with DALL-E 3.
+categories: 
+  - Market microstructure
+  - R
+  - data.table
+---
+
+Anyone active in market microstructure research *knows* that the devil is in the details. When clocks tick in microseconds and prices move in cents, a brief delay or a small fee discount can make a huge difference for traders. In fact, they can even be the business model of an exchange. But as much as such institutional detail is fascinating, the empirical implementation of even the most conventional microstructure concepts can be frustrating. Referring to the *interest of brevity*, many journal articles often defer the implementation details to an appendix, and even there they tend to be vague or incomplete. With the additional challenge of vast data sets, new entrants into this field face a steep challenge. 
```

Comment:
I left it as it is. Makes sense for me to highlight the paradox between "we all know that details matter" but at the same time we don't talk about them because of page limits.